t/ui/13-admin.t

```diff
@@ -677,17 +677,25 @@ subtest 'Manage API keys' => sub {
     };
 
     subtest 'create key with expiration date' => sub {
-        $driver->find_element('#api-keys-form input[type=submit]')->click;
-        is(scalar @{$driver->find_child_elements($tbody = api_keys_tbody, 'tr')}, 1, 'exactly one key present');
-        like($tbody->get_text, qr/\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}/, 'new key has expiration date');
+        my @rows = wait_for_element(
+            trigger_function => sub { $driver->find_element('#api-keys-form input[type=submit]')->click },
+            selector => '#api-keys-tbody > tr', description => 'key row present'
+        );
+        is(scalar @rows, 1, 'exactly one key present') or diag explain @rows;
+        like($rows[0]->get_text, qr/\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}/, 'new key has expiration date');
```

Comment:
`wait_for_element` will always return one value so this check doesn't make much sense and the variable name `@rows` is misleading. The `diag explain @rows` call will likely not print anything useful as well.

```suggestion
        my $row = wait_for_element(
            trigger_function => sub { $driver->find_element('#api-keys-form input[type=submit]')->click },
            selector => '#api-keys-tbody > tr', description => 'key row present'
        );
        ok defined $row, 'row with key present' or return;
        like $row->get_text, qr/\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}/, 'new key has expiration date';
```

Reply:
Ok, I combined the `wait_for_element` directly with `return` as the `wait_for_element` would already fail if it hasn't found an element.